### ISSUE TITLE
short all-in raise allowed further short raise

### ISSUE THREAD

**Issue by ctm** [ctm/mb2-doc#159]
```
Dealing 3000 6000 Omaha High/Low Eight-or-Better

KK blinds 1500
deadhead blinds 3000
1 B   JADC      17066    0  [  ] [  ] [  ] [  ]
2     KK        27467 1500  [  ] [  ] [  ] [  ]
3     deadhead  20000 3000  [  ] [  ] [  ] [  ]
4   > gerdog    37700    0  [  ] [  ] [  ] [  ]
5     chuck     18800    0  [  ] [  ] [  ] [  ]
6     GrizzB     3300    0  [  ] [  ] [  ] [  ]
7     NNR13     11167    0  [  ] [  ] [  ] [  ]
              My cards are: [5d] [2s] [As] [6c]
gerdog: heh
gerdog calls
chuck folds
GrizzB raises 300 to 3300 and is all-in
NNR13 folds
JADC calls
KK calls
It is 300 to you
deadhead raises 300 to 3600
```
I should not have had the option of raising 300. I should have had to complete the bet.

I should look at this when I look at #150

**Comment by ctm** [comment]
While I'm working on this ticket and short all-in raises reopening betting (#150) I should also limit the number of raises (#135).

In theory I'm not considering #135 high enough priority to put it on my Unsupervised Play-Testing milestone, but the lack of a limit has been off-putting in our nightly tournaments and it's also drop-dead simple, so might as well do it now.

**Comment by ctm** [comment]
I've decided to include #127 into the same branch with this one, #150 and #135. I think the end result will justify the delay. I don't like having four outstanding bugs for so long, but I've decided I want a struct that is used by the server to also be used by the client so that we can dynamically update the advanced action UI controls as action happens.

There's still a chance I'll change my mind again, but I _really_ want #86 tested and known good by Saturday, so now that I've written a bunch of notes to myself concerning these issues, I'll turn off my brain for a little while and plug away on my `player-tourneys-86` branch.

**Comment by ctm** [comment]
I bit the bullet and fixed #135 yesterday. In retrospect, I should have done that a long time ago, which has caused me to change my mind again, so I'll at least investigate a stop-gap solution to #150 and this ticket that is initially separate from #127.

If I can get this issue and #150 fixed first thing after I'm done exercising, then I can squash the other two (#172, #173) trivial bugs associated with the Unsupervised Play-testing milestone and get back to #86 before the end of my day, which would leave me fresh to finish it tomorrow, assuming no bugs or urgent feature requests are found in this evening's test.

**Comment by ctm** [comment]
Pathetic:
```
-            raise_amount
+            cmp::max(raise_amount, self.per_round.bet_minimum)
```
Fixed locally. Will be deployed with other fixes.
